system/core/base.php

```diff
@@ -321,15 +285,17 @@ function hypha_getHtml() {
 		$contents - html file contents
 	*/
 	function hypha_setHtml($contents) {
-		Hypha::$data->html->writeWithLock($contents);
+		global $O_O;
+		$O_O->data->html->writeWithLock($contents);
 	}
 
 	/*
 		Function: hypha_getCss
 		returns hypha base css file contents
 	*/
 	function hypha_getCss() {
-		return Hypha::$data->css->read();
+		global $O_O;
+		return $O_O->data->css->read();
```

Comment:
This should be `themeCss` in stead of `css`; `$O_O->data->css->read();`

Reply:
Fixed.